Utilities/Logging.cs

```diff
@@ -145,74 +110,22 @@ public static void incrementLogs()
             }
         }
 
-        internal static void Report(ErrorLevel errorLevel, string message, object data = null, [CallerMemberName] string memberName = "", [CallerFilePath] string filePath = "")
+        private static void Report(ErrorLevel errorLevel, string message, string data, string memberName, string filePath)
         {
-            Dictionary<string, object> thisData = PrepRollbarData(message, ref data);
-            if (thisData != null)
-            {
-                var rollbarReport = System.Threading.Tasks.Task.Run(() => SendToRollbar(errorLevel, message, data, thisData, memberName, filePath));
-            }
-        }
-
-        private static Dictionary<string, object> PrepRollbarData(string message, ref object data)
-        {
-            try
-            {
-                // It's not possible to scrub filepaths from exception messages, so since we don't want  
-                // to collect this personal data these exceptions need to be handled locally only.
-                if (data is Exception ex)
-                {
-                    if (ex.Message.Contains(Constants.DATA_DIR))
-                    {
-                        return null;
-                    }
-                }
-                else if (!(data is Dictionary<string, object>))
-                {
-                    var wrappedData = new Dictionary<string, object>()
-                    {
-                        {"data", data}
-                    };
-                    data = wrappedData;
-                }
-
-                // The Frontier API uses lowercase keys while the journal uses Titlecased keys. Establish case insensitivity before we proceed.
```

Comment:
Lines 179 - 198 are important for removing duplicate entries from the reported telemetry. We have a very limited budget when it comes to events per month and without these we will burn through items very quickly.

Reply:
I audited the calling paths and found none that were passing `data` as a dictionary rather than a string -- as evidenced by the fact that changing this method from `internal` to `private` gave no build errors.

If I've missed something then please yell.

Reply:
There are calling paths that pass data as a serialized dictionary.
And some which pass data in the message that instead should pass data in the data package where it can be de-duplicated.

Reply:
There are more of these than I want to enumerate, but please review the calling methods for Logging.cs `public static void Error(string message, string data = "", [CallerMemberName] string memberName = "", [CallerFilePath] string filePath = "")`

Reply:
On first review, do we still need to worry about these items? I'm not sure they get raised as errors in the first place any more?

Reply:
The entire purpose is to be able to respond effectively if they are raised, isn't it? And to be able to collect data about new items we need to add to our internal dictionaries. Just because they aren't being raised frequently now doesn't mean they won't be raised in the future (either because of a change we make or because of a change FDev makes).

Reply:
Example:
```
Logging.Error("Failed to map cargo type " + commodityName + " to commodity definition", line);
```
This sends a string to Rollbar that is readily serialized into an object where non-value added information like timestamps can be effectively removed by the current method. The PR would remove the data stripping and make each message non-unique. So the next time FDev introduces a new commodity, we might see hundreds or thousands of messages as users collect the new commodity. Just one would be sufficient.